Pulsarc/Utils/PulsarcDiscord.cs

```diff
@@ -9,26 +8,29 @@ static class PulsarcDiscord
     {
         private const string LOGO = "pulsarc_logo";
 
-        private static double lastUpdateTime = -15000;
+        private const double LastUpdateTime = -15000;
```

Comment:
This is misleading, `LastUpdateTime` is intended to change.

Reply:
this is part of closing everything amap